docs/examples/matplotlib_interact_example.rst

```diff
@@ -67,7 +67,7 @@ Example
        },
      }
    </script>
-   <script src="https://unpkg.com/thebe@latest/lib/index.js"></script>
+   <script src="../_static/lib/index.js"></script>
```

Comment:
It would simplify that path, as we would be loading from `../lib/index.js` as per the other pure HTML examples that we are copying.

In this case this example needs sphinx to compile it to HTML first -- I'm not clear on how or when to move the result of that build into the HTML though?